wiki/Ranking_Criteria/osu!mania/id.md

```diff
@@ -0,0 +1,199 @@
+
+#### Pedoman
+
+- **HP dan OD tidak boleh lebih dari 7.**
+- **Note snapping 1/4 berturut-turut atau lebih tidak boleh digunakan.** Ritme yang lebih lambat lebih cocok untuk tingkat kesulitan ini.
+- **Long note harus ditahan setidaknya 1 ketukan.** Waktu rilis yang lebih cepat terlalu sulit untuk pemain di level ini.
+- **Pelepasan long note setidaknya 1 ketukan satu sama lain.** Jarak rilis yang lebih pendek membutuhkan koordinasi tangan yang tidak dapat ditangani oleh pemain di level ini.
+- **Tidak boleh menempatkan objek selama durasi long note 1/1.** Namun, menempatkan objek di awal atau akhir tidak menjadi masalah.
+- **Kerapatan note harus terdiri sebagian besar 1/1, sesekali 1/2, atau untuk ritme yang lebih lambat.**
+- **Hindari penggunaan 5 kali note 1/2 secara berturut-turut.**
+- **Hindari penggunaan chord selama penggunaan 1/2 berturut-turut.** Namun, menggunakan chord untuk awal atau akhir tidak menjadi masalah.
+- **Hindari penggunaan chord lebih dari dua.** Ini termasuk long note yang melibatkan note lebih dari dua kolom.
+
+### ![](/wiki/shared/diff/normal-m.png) Normal
+
+#### Peraturan
+
+Jika tingkat kesulitan normal diperlukan dan digunakan sebagai *tingkat kesulitan terendah* dari sebuah beatmapset, maka harus mengikuti aturan berikut:
+
+- **Perubahan slider velocity yang mengubah scroll speed beatmap tidak diizinkan.** Pengecualian untuk perubahan slider velocity yang bertujuan untuk menyatukan scroll speed pada beatmap dengan variabel BPM.
+- **Normalisasi scroll harus dilakukan pada beatmap yang mempunyai beberapa BPM di dalamnya.**
+
+#### Pedoman
+
+- **HP dan OD tidak boleh lebih dari 7.5.**
+- **Note snapping 1/6 berturut-turut atau lebih tidak boleh digunakan.** Ritme yang lebih lambat lebih cocok untuk tingkat kesulitan ini.
+- **Long note harus ditahan setidaknya 1/2 ketukan.** Waktu rilis yang lebih cepat terlalu sulit untuk pemain di level ini.
+- **Pelepasan long note setidaknya 1/2 ketukan satu sama lain.** Jarak rilis yang lebih pendek membutuhkan koordinasi tangan yang tidak dapat ditangani oleh pemain di level ini.
+- **Tidak boleh menempatkan objek selama durasi long note 1/2.** Namun, menempatkan objek di awal atau akhir tidak menjadi masalah.
+- **Hindari penggunaan ritme yang lebih cepat dari 1/2 saat durasi long note 1/1.**
+- **Hindari menggunakan 1/4 shield.** Hal ini membutuhkan keseimbangan jari yang belum dapat ditangani oleh pemain di level ini.
+- **Kerapatan note harus terdiri sebagian besar 1/1 dan 1/2, sesekali 1/4, atau untuk ritme yang lebih lambat.**
+- **Hindari penggunaan 5 kali note 1/4 secara berturut-turut.**
+- **Stream panjang dengan snap 1/2 memerlukan, setidaknya 1/1 jeda untuk dapat diterapkan ke setiap frasa musik**, karena hal ini dapat menuntut fisik bagi pemain di level ini.
+- **Hindari penggunaan chord selama penggunaan 1/4 berturut-turut.** Namun, menggunakan chord untuk awal atau akhir tidak menjadi masalah.
```

Comment:
```suggestion
- **Hindari penggunaan chord selama pada note 1/4 berturut-turut.** Namun, menggunakan chord untuk awal atau akhir tidak menjadi masalah.
```

Reply:
Applied & remove "selama" for efficiency